server/views/components/sample/sample.njk

```diff
@@ -2,6 +2,6 @@
   <div class="sample-component">
     <h2 class="sample-component__heading">Sample component</h2>
     <p>{{ text }}</p>
-    {% include "views/components/grid-placeholder.njk" %}
+    {% include "views/grid/placeholder/placeholder.njk" %}
```

Comment:
the `placeholder/placeholder` duplication is strange, if we're going with `placeholder/readme` why not go with `placeholder/component`?